### PR TITLE
Load charts in Main Dashboard

### PR DESCRIPTION
**Pull request by MaximilianWagner**
Implement Load charts in Dashboard.
Backend integration for proper access.

Faster sampling with --test flag.

### CODE REVIEW

File: embark/templates/uploader/login.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js" type="text/javascript"></script>
 <script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
 
+
```
Comment (Vaish1795) [inline embark/templates/uploader/login.html:14]: Isn't it unnecessary?

**Review by Vaish1795 — COMMENTED** [review]
Looks clean :) good work :)

File: embark/templates/uploader/mainDashboard.html [hunk 2]
```diff
@@ -22,4 +22,75 @@
         <div class="card-body cardValue"></div>
     </div>
 </div>
+
+<div class="container">
+    <canvas id="loadChart"></canvas>
+  </div>
+
+  <script>
```
[the remark this answers is not in the document]
Comment (MaximilianWagner) [reply embark/templates/uploader/mainDashboard.html]: Export into mainDashboard.js and import as script.
Should be fixed.

File: embark/templates/uploader/login.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaximilianWagner) [reply embark/templates/uploader/login.html]: Yeah, my bad. ^^

**Pull request merged**